build/context.go

```diff
@@ -50,9 +50,11 @@ func (c *CloudBuilder) sendBuildContext() (err error) {
 	}()
 	// send to remote server
 	workdir := fmt.Sprintf(common.DefaultWorkDir, c.local.Cluster.Name)
-	if err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName); err != nil {
+	err = c.SSH.Copy(c.RemoteHostIP, tarFileName, tarFileName)
+	if err != nil {
 		return err
```

Comment:
We want return error details, for example:

return fmt.Errorf("failed to copy tar file: %s, err: %v", tarFileName, err)

This format is fine
```
 if err =xxx; err != nil {
...
}
```